app/views/games/_show_dev.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="container">
 <h1 style="font-weight:bold; margin-bottom: 20px; color: #EF7030;">Game Title: <%= @game.title %></h1>
-  <h3>General Ratings: <%= @game.reviews_count %> feedbacks</h3>
+  <h3>General Ratings: <%= @game.reviews.count %> feedbacks</h3>
```

Comment:
Make a method in the `Game` model called `reviews_count` that returns the reviews count